src/bootstrap/dist.rs

```diff
@@ -1401,6 +1405,10 @@ impl Step for Extended {
                 prepare("rls");
             }
 
+            if rustfmt_installer.is_some() {
+                prepare("rustfmt");
```

Comment:
Have you tested this? I don't believe the combined installer is configured to have this work.